community-edition/readme.md

```diff
@@ -17,36 +17,34 @@ Community Edition is designed for developers capable of working with the full Hu
 ## Prerequisites
 
 Before deploying the charts in this repo, you will need to choose and configure the following services...
```

Comment:
This line (Before deploying the charts...) should be moved to after the Node.js instructions.

Reply:
Assuming that the user is following the nodejs script and therefore is preparing to get the kubernetes configuration file (usually .yaml ) file which can be applied to the cluster